payment-request/user-abort-algorithm-manual.https.html

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
+<meta charset="utf8">
+<link rel="help" href="https://w3c.github.io/payment-request/#dom-paymentresponse-complete()">
```

Comment:
I think the link should be https://w3c.github.io/payment-request/#user-aborts-the-payment-request-algorithm.